ert_shared/ensemble_evaluator/evaluator.py

```diff
@@ -46,9 +46,7 @@ def __init__(self, ensemble, config, iter_, ee_id: str = "0"):
         self._done = self._loop.create_future()
 
         self._clients: Set[WebSocketServerProtocol] = set()
-        self._dispatchers_connected: asyncio.Queue[None] = asyncio.Queue(
-            loop=self._loop
-        )
+        self._dispatchers_connected: asyncio.Queue[None] = asyncio.Queue()
```

Comment:
The problem here is that when you remove the loop here it uses the current event loop at the time when the EnsembleEvaluator is instantiated, however we want this to be the event loop from the thread that is running websocket.  So we might need to create all of these things inside that thread instead